Makefile

```diff
@@ -63,11 +63,6 @@ coderd/database/dump.sql: $(wildcard coderd/database/migrations/*.sql)
 coderd/database/querier.go: coderd/database/sqlc.yaml coderd/database/dump.sql $(wildcard coderd/database/queries/*.sql)
 	coderd/database/generate.sh
 
-# This target is deprecated, as GNU make has issues passing signals to subprocesses.
-dev:
-	@echo Please run ./scripts/develop.sh manually.
-.PHONY: dev
-
```

Comment:
@johnstcn I haven't worked much with `Makefile`s. Does this look right? Did I miss anything?

Reply:
Looks fine to me!